### PR TITLE
add a few packages

### PR DESCRIPTION
**Pull request by daattali**
Added a few packages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +165,12 @@ A curated list of resources for R Shiny.  (*Featured on [Awesome-R](https://awes
 - [reactlog](https://github.com/rstudio/reactlog) - Visual insight into a Shiny apps reactive graph.
 - [supreme](https://github.com/strboul/supreme) - Structure Shiny applications developed with modules.
 - [shinymanager](https://github.com/datastorm-open/shinymanager/) - Simple and secure authentication for single 'Shiny' applications using a SQLite database.
+- [shinyTime](https://github.com/burgerga/shinyTime) - A timeInput widget for Shiny
+- [shinyalert](https://github.com/daattali/shinyalert) - Easily create pretty popup messages (modals) in Shiny
+- [shinydisconnect](https://github.com/daattali/shinydisconnect) - Show a nice message when a Shiny app disconnects or errors
+- [shinyscreenshot](https://github.com/daattali/shinyscreenshot) - Capture screenshots of entire pages or parts of pages in Shiny apps
+- [reactor](https://github.com/yonicd/reactor) - Unit testing for shiny reactivity
+- [flextable](https://github.com/davidgohel/flextable) - Easily create customizable tables
```
Comment (grabear) [inline README.md]: Flextable is an awesome addition.  I haven't had the opportunity to use it, but I'm working on a report right now so I will definitely give it a try.  

I think this belongs in the HTML-widgets section though https://github.com/grabear/awesome-rshiny#integrations

Comment (grabear) [inline README.md]:
```suggestion
```

**Review by grabear — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (daattali) [reply README.md]: Ah yeah that makes sense!

**Review by grabear — APPROVED** [review]

**Comment by grabear** [comment]
Thanks @daattali.  Sorry for the late update on this.  Hope you are well!

**Pull request merged**